src/test/scala/com/cognite/sdk/scala/common/FilterTest.scala

```diff
@@ -46,7 +50,7 @@ class FilterTest extends SdkTestSpec {
       auth)(requestHijacker)
     val dummyRequestSession = dummyClient.requestSession
 
-    Filter.filterWithCursor(
+    val _ = Filter.filterWithCursor(
```

Comment:
This is new to me. Just to highlight that this is returning something we don't use?

Reply:
Yes, a lot like https://www.baeldung.com/scala/underscore#2-ignored-variable, but not used in a pattern here.